sale_quotation_number/models/sale_order.py

```diff
@@ -4,12 +4,14 @@
 # © 2020 Manuel Regidor  <manuel.regidor@sygel.es>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class SaleOrder(models.Model):
     _inherit = "sale.order"
 
+    confirmed_before = fields.Boolean(copy=False)
```

Comment:
@aaladro-kernet Thanks for this. I would say this is a little bit too generic. Maybe `quotation_number_set` or something like that.

Reply:
Moreover, a little test and a migration script (that set that field for 'SQ' quotations) are welcome

Reply:
This field is simply to control the order, once confirmed, go back to draft and reconfirm. Do not assign a new sequence number. Are we agreed?